src/Recipe.php

```diff
@@ -70,6 +71,24 @@ public function isContrib(): bool
         return $this->data['is_contrib'] ?? false;
     }
 
+    public static function queueCompare(Recipe $first, Recipe $second): int
```

Comment:
- instead of a generic mechanism, I think a special case for framework-bundle only could be enough
- this means we should also remove the recipe for symfony/flex, which is useless anyway, since framework-bundle is a mandatory dep
- which means we could "just" deal with that case in `Flex::fetchRecipes()`, wdyt?

Reply:
see https://github.com/symfony/recipes/pull/295

Reply:
@nicolas-grekas if there will be only `symfony/framework-bundle` recipe, then we could only prepend "install FrameworkBundle" operation to `$this->operations` instead of appending it in [`Symfony\Flex\Flex::record`](https://github.com/symfony/flex/blob/8917eb9e7d490088b85ea2a4cd7f2ad52bb0d9ee/src/Flex.php#L117)